esengine/mapping.py

```diff
@@ -45,3 +51,38 @@ def save(self, es=None):
                 index=self.document_class._index,
                 body=self.generate()
             )
+
+    def create_all(self, models_to_mapping, custon_settings, es=None):
+        """
+        Add custon settings like filters and analizers to index.
+
+        Add custon settings, like filters and analizers, to index. Be aware
+        that elasticsearch only allow this operation on index creation.
+        """
+        if not isinstance(models_to_mapping, collections.Iterable):
+            raise AttributeError('models_to_mapping must be iterable')
+
+        mapped_models = [x for x in models_to_mapping]
+        if custon_settings:
+            indexes = set()
+            for model in mapped_models:
+                indexes.add(model._index)
+                if es is None:
```

Comment:
aqui nao precisa desse "if" apenas ``es = model.get_es(es)`` se o es for != de None o get_es nao fara nada, apenas tentara pegar o _es caso o es passado seja None